src/sections/explain-sponsoring/explain-sponsoring.js

```diff
@@ -1,10 +1,10 @@
 import React from "react"
 import cn from "classnames"
 
-import Section from "../section"
-import Spacer from "../spacer"
-import TextBlock from "../text-block"
-import Typography from "../typography"
+import Section from "../../components/section";
```

Comment:
Can you check prettier settings? The imports above don't have a `;`, these here have it

Reply:
Good you saw this. 
We have no prettier as a pre commit hook, and I didn't had it activated...
But would you like to run a 
`yarn format`
?